### ISSUE TITLE
Edge support documentation

### ISSUE THREAD

**Issue by paulfalgout** [cypress-io/circleci-orb#419]
The documentation for edge is either incorrect or confusing.  The example shows
```yaml
version: '2.1'
orbs:
  cypress: cypress-io/cypress@3
workflows:
  use-my-orb:
    jobs:
      - cypress/run:
          cypress-command: npx cypress run --browser edge
          install-browsers: true
          start-command: 'npm run start:dev'
```

But the `install-browsers` command description says:
```
Cypress runs by default in the Electron browser. Use this flag to install additional browsers to run your tests in. This is only needed if you are passing the `--browser` flag in your `cypress-command`. This parameter leverages the `circleci/browser-tools` orb and includes Chrome and FireFox. If you need additional browser support you can set this to false and use an executor with a docker image that includes the browsers of your choosing.
```

This seems to indicate that `install-browsers` should be `false` for edge and we should load an `executor`?  Is the Edge example incorrect or am I missing something?

**Comment by emilyrohrbough** [comment]
@paulfalgout The example you shared is incorrect example-- can can you share where you found this example? 'm finding a few references:
- https://github.com/cypress-io/circleci-orb/blob/master/src/examples/edge.yml
  - Mostly correct - `install-browsers: true` parameter is unneeded since it will only install Chrome or Firefox per the description.
- (incorrect - missing ) https://github.com/cypress-io/circleci-orb/blob/master/src/examples/browser.yml
  - Incorrect - `install-browsers: true` parameter is unneeded  since it will only install Chrome or Firefox per the description. The `executor` (missing) needs to be specified with the correct docker image with edge pre-installed or a custom-step to install edge is needed.

**Comment by emilyrohrbough** [comment]
Ah I see this is out of date: https://circleci.com/developer/orbs/orb/cypress-io/cypress#usage-edge

**Comment by emilyrohrbough** [comment]
This has been fixed with the 3.1.0 release.

**Comment by MikeMcC399** [comment]
- The `circleci-orb` release [v3.1.0](https://github.com/cypress-io/circleci-orb/releases/tag/v3.1.0) included PR #420 with an update to [src/examples/edge.yml](https://github.com/cypress-io/circleci-orb/blob/master/src/examples/edge.yml). This example however was not published and it fails to run.

> Can't run because you've entered an invalid browser name.
>
> Browser: edge was not found on your system or is not supported by Cypress.

**Comment by MikeMcC399** [comment]
- Follow-on PR https://github.com/cypress-io/circleci-orb/pull/489
